src/test/resources/related-objects/correction-requests/update-request-partial.json

```diff
@@ -1,3 +1,3 @@
 {
-  "descriptionOfChange": "Description needs to mention prisoner numbers"
+  "descriptionOfChange": "Description needs to mention prisoner name"
```

Comment:
annoyingly, the partial update test case is basically the same as the normal one.
…but it'd take a bigger change to the abstract test classes to make partial update test cases optional for related entities.